core/modules/css/CssAccess.cc

```diff
@@ -467,74 +467,16 @@ MatchTableParams
 CssAccess::getMatchTableParams(std::string const& dbName,
                                std::string const& tableName) const {
     LOGS(_log, LOG_LVL_DEBUG, "getMatchTableParams(" << dbName << ", " << tableName << ")");
-    _checkVersion();
-
-    std::string const tableKey = _prefix + "/DBS/" + dbName + "/TABLES/" + tableName;
-
-    MatchTableParams params;
-
-    std::vector<std::string> subKeys{"dirTable1", "dirColName1", "dirTable2", "dirColName2", "flagColName"};
-    auto paramMap = _getSubkeys(tableKey + "/match", subKeys);
-    if (paramMap.empty()) {
-        // check table key
-        if (not _kvI->exists(tableKey)) throw NoSuchTable(dbName, tableName);
-        return params;
-    }
-
-    // fill the structure
-    params.dirTable1 = paramMap["dirTable1"];
-    params.dirColName1 = paramMap["dirColName1"];
-    params.dirTable2 = paramMap["dirTable2"];
-    params.dirColName2 = paramMap["dirColName2"];
-    params.flagColName = paramMap["flagColName"];
-
-    return params;
+    TableParams params = getTableParams(dbName, tableName);
+    return params.match;
```

Comment:
OK, I have already forgotten that I wanted to deprecate those two methods. Still it does not matter, until we actually remove them I propose to keep things as they are now. Implementing `getTableParams()` in terms of two other methods is incorrect (it runs in two separate queries which is inefficient and potentially inconsistent). Your original idea (implementing two other methods based on `getTableParams()`) is better but has other issues as I mentioned above. 

Actually deprecating those two methods and removing them needs significant changes in czar and I do not know how soon it's going to happen, again, until we are ready to do that I suggest we keep with the current approach.